### DISCUSSION TITLE
Version Catalog instead of `Versions.kt` in `buildSrc`

### DISCUSSION THREAD

**Discussion by mervyn-mccreight** [failgood/failgood#403] [General]
Hey :)

I wanted to ask if anything speaks against using the [version catalog](https://docs.gradle.org/current/userguide/platforms.html) feature of Gradle instead of the `Versions.kt` in `buildSrc`.

IMO it's great that Gradle provides a proper standard way of defining versions in a single place. So my thoughts were like - maybe it's a good idea to use the standard approach of Gradle to make it easier for possible contributors to understand the build/the project setup.

WDYT?

**Comment by mervyn-mccreight** [comment]
Renovate seems to support it (at least the `TOML`-variant) out of the box: https://docs.renovatebot.com/modules/manager/gradle/

**Reply by christophsturm** [reply]
Can you make a PR and then we look at the pros and cons?

**Reply by mervyn-mccreight** [reply]
Sure, will do

**Reply by christophsturm** [reply]
but don't invest too much work into it, i think we should only do it if it really is an improvement.
